src/Resources/config/doctrine/PageTranslation.orm.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                                      http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
+    <mapped-superclass name="MonsieurBiz\SyliusCmsPagePlugin\Entity\PageTranslation" table="monsieurbiz_cms_page_translation">
+        <id name="id" type="integer">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="title" length="255" nullable="true" />
+        <field name="content" type="text" nullable="true" />
+        <field name="slug" length="255" nullable="true" />
+        <field name="metaTitle" length="255" nullable="true" />
+        <field name="metaKeywords" length="255" nullable="true" />
+        <field name="metaDescription" type="text" nullable="true" />
```

Comment:
Could you explain why the locale field is missing?

I think it's not a mistake but I don't understand why yet.

Reply:
That's because we extend from AbstractTranslation which has already this definition.